src/views/DatasetManager/components/DatasetOverview/components/DatasetOverviewPlaceholder/DatasetOverviewPlaceholder.js

```diff
@@ -102,6 +104,18 @@ export const DatasetOverviewPlaceholder = () => {
       <ReuploadFileDatasetButton datasetId={currentDatasetId} iconButton={false} />
     ) : (
       <>
+        {!Object.values(DATASET_SOURCE_TYPE).includes(currentDatasetType) && (
+          <Button
+            sx={{ mr: 1 }}
+            data-cy={'successful-run-logs-download-button'}
```

Comment:
Can we change the data-cy here? This button is not used for "succesful runs" :slightly_smiling_face: